txpipe/data_types/types.py

```diff
@@ -396,3 +396,19 @@ def to_ccl(self, **kwargs):
         inits.update(kwargs)
 
         return ccl.Cosmology(**inits)
+
+class FitsFile(DataFile):
```

Comment:
`FitsFile` already exists in the base data classes: https://github.com/LSSTDESC/TXPipe/blob/HSC/txpipe/data_types/base.py#L214